client/jetpack-cloud/sections/pricing/header/index.tsx

```diff
@@ -4,6 +4,7 @@
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import React, { useMemo } from 'react';
+import moment from 'moment';
```

Comment:
I think it's better to use `useLocalizedMoment` since it provides some context for localization already.

Reply:
That only deals with date formatting though, right?

Reply:
Maybe. I'm not sure if it affects what's bundled as well.